src/Microsoft.AspNetCore.Routing/Matchers/MatcherPolicy.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Routing
+{
+    public abstract class MatcherPolicy
```

Comment:
I considered also making this a marker interface. We need to define some kind of type because DI.